__tests__/app.test.js

```diff
@@ -125,3 +126,39 @@ describe("CORE: GET - /api/articles/:article_id", () => {
         });
         //write error case for 0 results : 404
     });
+
+
+  describe("CORE: GET - /api/articles", () => {
+    test("200: Endpoint should contain all article objects except body", () => {
+        return request(app)
+          .get("/api/articles")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles.length).not.toBe(0)
+       
+            body.articles.forEach((article) => {
+              expect(article).toHaveProperty("article_id", expect.any(Number));
+              expect(article).toHaveProperty("title", expect.any(String)); 
+              expect(article).toHaveProperty("topic", expect.any(String)); 
+              expect(article).toHaveProperty("author", expect.any(String)); 
+              expect(article).toHaveProperty("created_at", expect.any(String)); 
+              expect(article).not.toHaveProperty("body"); 
+              expect(article).toHaveProperty("votes", expect.any(Number)); 
+              expect(article).toHaveProperty("article_img_url", expect.any(String))
+              expect(article).toHaveProperty("comment_count", expect.any(Number));; 
+            });
+          });
+      });
+      test("200: Articles are correctly ordered (Descending by Date Created)", () => {
+        return request(app)
+          .get("/api/articles")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles.length).not.toBe(0)
+            const dates = body.articles.map((article) => {
+              return article.created_at
+            })        
+            expect(dates).toBeSorted({ descending: true })
+          });
+      });
```

Comment:
These tests are for getAllArticles but have been included in a PR for ticket 6? 

A little confused by this?